mmdet/models/dense_heads/base_dense_head.py

```
@@ -484,6 +484,14 @@ def _bbox_post_process(self,
             bboxes = get_box_tensor(results.bboxes)
             det_bboxes, keep_idxs = batched_nms(bboxes, results.scores,
                                                 results.labels, cfg.nms)
+            # Ensure indices are accepted by mmengine InstanceData across devices
+            try:
+                keep_idxs = keep_idxs.tolist()
+            except Exception:
+                try:
+                    keep_idxs = list(keep_idxs)
+                except Exception:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)